src/implementations/twig/components/breadcrumb/breadcrumb.html.twig

```diff
@@ -0,0 +1,145 @@
+{% spaceless %}
+
+{#
+  Parameters:
+    - "variant" (string) options:[ negative ] (default: '') Negative style with a darker background (only for EC)
+    - "links" (array) (default: []): format: [
+      {
+        "label" (string) (default: '') Label of link
+        "path" (string) (default: '') Url of link
+      },
+      ...
+    ],
+    - "navigation_text" (string) (default: ''): Text of navigation in breadcrumb
+    - "ellipsis_label" (string) (default: ''): Label of the ellipsis, e.g. "Click to expand"
+    - "ellipsis_text" (string) (default: '...'): Text of the ellipsis
+    - "icon_file_path" (string) (default: ''): URL to icons file
+    - "icon_size" (string) (default: '2xs'): Size of the icon between segments
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (optional) (string)
+        ...
+      ]
+#}
+
+{# Internal properties #}
+
+{% set _variant = variant|default('') %}
+{% set _breadcrumb_size = links|length %}
+{% set _links = links|default([]) %}
+{% set _icon_file_path = icon_file_path|default('') %}
+{% set _icon_size = icon_size|default('2xs') %}
+{% set _navigation_text = navigation_text|default('') %}
+{% set _ellipsis_label = ellipsis_label|default('') %}
+{% set _ellipsis_text = ellipsis_text|default('…') %}
+{% set _extra_attributes = '' %}
+{% set _breadcrumb_segment_class = 'ecl-breadcrumb__segment' %}
+{% set _min_items_left = min_items_left|default(1) %}
+{% set _min_items_right = min_items_right|default(2) %}
+{% set _nav_attributes = [
+  { name: 'aria-label', value: _navigation_text },
+  { name: 'data-ecl-breadcrumb', value: 'true' }]
+%}
+{% set _breadcrumb_icon = {
+  icon: {
+    name: 'corner-arrow',
+    size: _icon_size,
+    transform: 'rotate-90',
+    path: _icon_file_path,
+  },
+  extra_classes: 'ecl-breadcrumb__icon',
+  extra_attributes: [{ name: 'role', value: 'presentation' }]
+} %}
+
+{% set _css_class = 'ecl-breadcrumb' %}
+
+{# Internal logic - Process properties #}
+
+{% if _variant is not empty and _variant == 'negative' %}
```

Comment:
do we really need to test if the variant is "negative"? we could just append to the class name, that's what is done in most components